discovery.md

```diff
@@ -136,6 +136,7 @@ Service:
   "url": "[unique URL to this service]",
   "description": "[human string]", ?
   "docsurl": "[URL reference for human documentation]", ?
+  "removaltime": "[RFC339 timestamp of when service will be removed]", ?
```

Comment:
Shouldn't we give guidance on what to migrate to?
```suggestion
  "removal": {
    "time": "[RFC339 timestamp of when service will be removed]", ?
    "alternative": "", ?
  }
```

Reply:
Questions from the group:

John:
- what kind of guarantees are there around "alt"?  does it have it to exist before the time?
- do subscriptions carry over to the alt?

- Erik: any kind of transition?

Jennifer:
- what's the value? id or url? machine or human readable?
- or how does someone interpret it?

Reply:
My recommendation, if we choose to include "alt":
- the "alt" must be available immediately 
- subscriptions do not carry over and there's no guarantee that the 'alt' is a semantic replacement for the service. It is simply a recommendation for consideration and the user is expected to investigate if it's a good match.
- no transitioning or transferring of subscriptions to "alt"
- "alt" must be the URL of the alt service. MAY be in another DE, but no guarantee that the client can access it.

Reply:
s/removal/deprecated/